.travis.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
How about we get this script from a common location instead of copy-pasting it to each repository?
@OriHoch

Reply:
sure, `curl` it from budgetkey-k8s ?